### PR TITLE
Followup to #2456: update examples, add trust

### PR DESCRIPTION
**Pull request by edsantiago**
- belatedly incorporate review feedback from baude
- add usage synopsis for trust-set and trust-show

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
@baude sorry I slept in this morning

**Comment by rhatdan** [comment]
LGTM

File: cmd/podman/checkpoint.go [hunk 1]
```diff
@@ -32,9 +32,9 @@ var (
 		Args: func(cmd *cobra.Command, args []string) error {
 			return checkAllAndLatest(cmd, args, false)
 		},
-		Example: `podman checkpoint --keep ctrID
-  podman checkpoint --all
-  podman checkpoint --leave-running --latest`,
+		Example: `podman container checkpoint --keep ctrID
+  podman container checkpoint --all
+  podman container checkpoint --leave-running --latest`,
```
Comment (TomSweeneyRedHat) [inline cmd/podman/checkpoint.go]: oops, ty for the fix up!

File: cmd/podman/exists.go [hunk 2]
```diff
@@ -65,7 +65,7 @@ var (
 			podExistsCommand.GlobalFlags = MainGlobalOpts
 			return podExistsCmd(&podExistsCommand)
 		},
-		Example: `podman pod exists podID`,
+		Example: `podman pod exists mypod || podman pod create --name mypod`,
 	}
```
Comment (TomSweeneyRedHat) [inline cmd/podman/exists.go]: I'd suggest keeping the original examples here for these three exists commands and adding the ones that you've worked up as the second example for each.  FW

Comment (edsantiago) [reply cmd/podman/exists.go]: That's easy to do; but first, @baude, could you provide a little more guidance on your review comment of #2456? What sort of examples were you hoping to see?

**Comment by mheon** [comment]
LGTM once Tom's comments are addressed

**Comment by TomSweeneyRedHat** [comment]
ubuntu failing here too...

**Pull request merged**